package.json

```diff
@@ -4,18 +4,19 @@
   "private": true,
   "scripts": {
     "build-all": "yarn run build-ui && yarn run build-css",
-    "postinstall": "bower install",
-    "clean": "rm -rf output bower_components node_modules && cd css && rm -rf node_modules",
-    "build-ui": "yarn run pulp -- build -I ui-guide --to dist/example.js -- $RTS_ARGS",
-    "watch-ui": "yarn run pulp -w --then 'yarn run build-css' build -I ui-guide --to dist/example.js",
+    "postinstall": "",
+    "setup": "npm i && cd css && npm i",
```

Comment:
No theres nothing special about `setup`, its just a standard script. The problem with putting it in `postInstall` is it will cause the `install node/spago dependencies" to run longer even though there's the dependencies will already have been added.

Maybe a better alternative is to add everything in the `setup` script to the `clean` script. That way clean won't actually clobber out all the npm packages. What do you think? @davezuch

Reply:
Haha the latter - didn't want to increase the build-time on circle ci any further than it already has. If that doesn't seem like an issue, I can just throw it in postinstall. A better build system will help in the future anyhow

Reply:
Got it, let's throw it in for now and see if the increased build time is worth the savings in not having to remember to install the css separately. At least from the perspective that we'll improve the build system soon.